ipynb/sleep_apnea/SleepApnea-qiime2-tutorial.ipynb

```diff
@@ -1,54 +1,190 @@
+      "  --p-rank INTEGER                The underlying low-rank structure\r\n",
+      "                                  (suggested: 1 < rank < 10)  [default: 3]\r\n",
+      "  --p-min-sample-count INTEGER    Minimum sum cutoff of sample across all\r\n",
+      "                                  features  [default: 500]\r\n",
+      "  --p-min-feature-count INTEGER   Minimum sum cutoff of features across all\r\n",
+      "                                  samples  [default: 10]\r\n",
+      "  --p-iterations INTEGER          The number of iterations to optomize the\r\n",
+      "                                  solution (suggested to below 100, beware of\r\n",
+      "                                  overfitting)  [default: 5]\r\n",
+      "  --o-biplot ARTIFACT PATH PCoAResults % Properties(['biplot'])\r\n",
+      "                                  A biplot of the (Robust Aitchison) RPCA\r\n",
+      "                                  feature loadings  [required if not passing\r\n",
+      "                                  --output-dir]\r\n",
+      "  --o-distance-matrix ARTIFACT PATH DistanceMatrix\r\n",
+      "                                  The Aitchison distance of the sample\r\n",
+      "                                  loadings from RPCA.  [required if not\r\n",
+      "                                  passing --output-dir]\r\n",
+      "  --output-dir DIRECTORY          Output unspecified results to a directory\r\n",
+      "  --cmd-config FILE               Use config file for command options\r\n",
+      "  --verbose                       Display verbose output to stdout and/or\r\n",
+      "                                  stderr during execution of this action.\r\n",
+      "                                  [default: False]\r\n",
+      "  --quiet                         Silence output if execution is successful\r\n",
+      "                                  (silence is golden).  [default: False]\r\n",
+      "  --citations                     Show citations and exit.\r\n",
+      "  --help                          Show this message and exit.\r\n"
      ]
     }
    ],
    "source": [
-    "!qiime dev refresh-cache"
+    "! qiime deicode rpca --help"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "First we will convert the biom file to a Qiime2 style qza table"
+    "The first are sample filtering cutoffs these are  p-min-feature-count and p-min-sample-count. Both of these parameters accept integer values and remove feature or samples, respectively, with sums below this cutoff. The feature cut-off is useful in the case that features with very low total counts among all samples represent contamination or chimeric sequences. The sample cut off is useful for the case that some sample received very few reads relative to other samples.\n",
+    "\n",
+    "**Note:** it _not_ recommended to bin your features by taxonomic assignment (i.e. by genus level). <br>\n",
```

Comment:
it not -> it is not (same for next line).

Reply:
fixed, thank you!